src/google_cloud_debugger/google_cloud_debugger_lib/dbg_enum.cc

```diff
@@ -43,60 +45,46 @@ HRESULT DbgEnum::PopulateValue(Variable *variable) {
     return S_OK;
   }
 
-  CorElementType enum_type;
-  bool enum_type_found = false;
-  // We have enum type.
-  // First, gets the underlying type. This is from the non-static field __value.
-  for (auto it = begin(class_fields_); it != end(class_fields_); ++it) {
-    if (*it) {
-      if ((*it)->IsStatic()) {
-        continue;
-      }
-
-      if (kEnumValue.compare((*it)->GetMemberName()) != 0) {
-        continue;
-      }
-
-      PCCOR_SIGNATURE field_signature = (*it)->GetSignature();
-      enum_type = CorSigUncompressElementType(field_signature);
-      enum_type_found = true;
-      break;
-    }
-  }
-
-  if (!enum_type_found) {
+  if (enum_type_ == CorElementType::ELEMENT_TYPE_END) {
```

Comment:
I'm a little confused about this.  [ELEMENT_TYPE_END 'Used internally'](https://docs.microsoft.com/en-us/dotnet/framework/unmanaged-api/metadata/corelementtype-enumeration).  Can you add some context here?